osbuild/testutil/net.py

```diff
@@ -25,6 +30,16 @@ def __init__(self, *args, **kwargs):  # pylint: disable=unused-argument
 from .atomic import AtomicCounter
 
 
+def list_dir(directory):
```

Comment:
I wonder if we could use `os.walk` here as well ?

Reply:
Sorry what?

Reply:
`os.walk` is the built-in for recursively iterating over directory contents.

Reply:
Anyway, I am going to rename this to `print_dir` instead, does not return a list this is misleading.